### DISCUSSION TITLE
Convert my username to lowercase

### DISCUSSION THREAD

**Discussion by gitjit** [community/community#23757] [New to GitHub]
<p>How can I convert my GitHub username to lowercase?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>Unfortunately because it’s treated by our system as the same name, you can either:</p>
<ul>
<li>Change your username to something else entirely and then change it back to the lowercase version. Username changes are typically more or less instant.</li>
</ul>
<p>or</p>
<ul>
<li>If you worry your username might get nabbed while you are making the switch, you can get in touch with support and we can make the change directly. Please note: this will require more patience than the DIY option :wink:
</li>
</ul>
<p><a href="https://support.github.com/contact" class="onebox" target="_blank">https://support.github.com/contact</a></p>
